components/admin_console/brand_image_setting/brand_image_setting.jsx

```diff
@@ -22,20 +21,35 @@ export default class BrandImageSetting extends React.PureComponent {
          * Set to disable the setting
          */
         disabled: PropTypes.bool.isRequired,
+
+        /*
+        * Set the save needed in the admin schema settings to trigger the save button to turn on
+        */
+        setSaveNeeded: PropTypes.func.isRequired,
+
+        /*
+        * Registers the function suppose to be run when the save button is pressed
+        */
+        registerSaveAction: PropTypes.func.isRequired,
+
+        /*
+        * Unregisters the function on unmount of the component suppose to be run when the save button is pressed
+        */
+        unRegisterSaveAction: PropTypes.func.isRequired,
     }
 
     constructor(props) {
         super(props);
 
         this.handleImageChange = this.handleImageChange.bind(this);
```

Comment:
Not a stopper, but we normally prefer to use "arrow functions" to avoid the need of binding this, for example if you define the `handleImageChange` in the component as:
```js
    handleImageChange = () => {
        ...
    }
```
instead of 
```js
    handleImageChange() {
        ...
    }
```

You wont need the binding.